tasks/main.yml

```diff
@@ -7,3 +12,12 @@
     group: root
     mode: 0644
   notify: Restart systemd-sysctl
+
+- name: Template sysfs config
+  template:
+    src: sysfs.conf.j2
+    dest: "/etc/sysfs.d/{{ sysfs_number }}-ansible.conf"
```

Comment:
```suggestion
    dest: /etc/sysfs.d/{{ sysfs_number }}-ansible.conf
```

Reply:
I wanted to keep this similar to the other task (`Template sysctl config`)